Source/WebInspectorUI/UserInterface/Views/ComputedStyleSection.js

```diff
@@ -129,16 +130,38 @@ WI.ComputedStyleSection = class ComputedStyleSection extends WI.View
         this.needsLayout();
     }
 
+    get isEmpty()
+    {
+        this.updateLayoutIfNeeded();
```

Comment:
oof is there no way to avoid this?  we shouldn't do sync forced layout unless absolutely necessary